src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue24741.cs

```diff
@@ -0,0 +1,52 @@
+﻿#if WINDOWS
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue24741 : _IssuesUITest
+	{
+		public Issue24741(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "Unable to select tab after backing out of page and returning";
+
+		[Test]
+		[Category(UITestCategories.TabbedPage)]
+		public void SelectTabAfterNavigation()
```

Comment:
Upload screen shots

Reply:
@PureWeen, I have updated Screenshot for this issue. Could you please validate it once and let me know if there are any further concerns.